content/en/docs/instrumentation/erlang/getting-started/rebar.md

```diff
@@ -1,12 +1,9 @@
 ---
-title: Getting Started
-weight: 20
+title: Rebar3 / Mix
+aliases: [/docs/erlang/getting_started/rebar]
```

Comment:
No need for an alias:

```suggestion
```

Reply:
thanks! totally just cargo-culted these from the other doc